cmd/apps/vault_app.go

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) arkade author(s) 2022. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package apps
+
+import (
+	"fmt"
+
+	"github.com/alexellis/arkade/pkg"
+	"github.com/alexellis/arkade/pkg/apps"
+	"github.com/alexellis/arkade/pkg/config"
+	"github.com/alexellis/arkade/pkg/types"
+	"github.com/spf13/cobra"
+)
+
+func MakeInstallVault() *cobra.Command {
+	var vault = &cobra.Command{
+		Use:          "vault",
+		Short:        "Install vault",
+		Long:         `Install vault`,
+		Example:      `  arkade install vault`,
```

Comment:
Can you give an example with ingress and domain please?

Reply:
@Shikachuu @Jasstkn @welteki @rgee0 we can probably improve the `example` field for the various apps that we like to use from arkade on a regular basis with an example or two.

Would you each take just one and see what you can do?

Reply:
Sure, I can take `traefik`, `redis`, `grafana` and `nfs-provisioner` in the first round, since I use this almost daily with `--set` flags.

Reply:
I'll check about it. 
For now result is the following:
```
Install vault

Usage:
  arkade install vault [flags]

Examples:
  arkade install vault
  # with ingress and custom domain
  arkade install vault --ingress=true --domain=vault.example.com
```